src/components/IconButton.tsx

```diff
@@ -107,10 +121,15 @@ export function IconButton(props: IconButtonProps) {
 
 const defaultIconColor = Palette.Gray900;
 const contrastIconColor = Palette.White;
+const circleIconColor = Palette.Gray700;
 const iconButtonStylesReset = Css.bcTransparent.bss.bgTransparent.cursorPointer.outline0.dif.aic.jcc.transition.$;
 const iconButtonNormal = Css.hPx(28).wPx(28).br8.bw2.$;
 const iconButtonCompact = Css.hPx(18).wPx(18).br4.bw1.$;
+const iconButtonCircle = Css.br100.wPx(48).hPx(48).bcGray300.ba.bw1.df.jcc.aic.onHover.bgBlue100.bcBlue200.$;
 export const iconButtonStylesHover = Css.bgGray200.$;
 export const iconButtonContrastStylesHover = Css.bgGray700.$;
+export const iconButtonCircleStylesHover = Css.bgBlue100.bcBlue200.$;
```

Comment:
Question: any reason these are not set via the `isHovered` above?

Reply:
Initially, I thought that all the ternaries were confusing and it seemed more straightforward to add the `onHover` on the `iconButtonCircle`, but then I added this `iconButtonCircleStylesHover` to use in the story. I agree that it doesn't make sense to have this in 2 places, so I can add `iconButtonCircleStylesHover` to the `isHovered` instead.